.github/actions/generate-resources/action.yml

```diff
@@ -1,5 +1,5 @@
-name: 'generate resources'
+name: 'Generate resources'
 description: 'Read Developer DAO Airtable and generate README.md'
 runs:
   using: 'node12'
-  main: 'main.js --run'
\ No newline at end of file
+  main: 'init.js'
```

Comment:
Removed `--run` because I couldn't figure out how to support flags, but still support testing. So I abstracted the code from `main.js` to `init.js` so that it can still support jest doing tests on various functions in `main.js`